src/main/java/spoon/reflect/factory/Factory.java

```diff
@@ -321,6 +326,11 @@
 	 */
 	List<CtExpression<?>> createVariableReads(List<? extends CtVariable<?>> variables);
 
+	/**
+	 *  @see CodeFactory#createVariableWrites(List)
+	 */
+	List<CtExpression<?>> createVariableWrites(List<? extends CtVariable<?>> variables);
+
```

Comment:
For what usages do we need to create a list of `CtVariableWrite`?
(I assume the list of `CtVariableRead` is useful when passing them as parameters to a method invocation.)

Reply:
Right. I just added it to keep methods in right polarity.
I can remove it if you think that there is no need to implement it.